database/migrations/2023_01_20_211900_add_member_until_to_member.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddMemberUntilToMember extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('members', function (Blueprint $table) {
+            $table->bigInteger('member_until')->after('is_pending')->nullable();
```

Comment:
Naming the column `member_until` in the `members` table seems redundant, maybe name the column `until` instead.